src/support/tslib.ts

```diff
@@ -0,0 +1,41 @@
+import global from '../global';
+
+import 'tslib';
+
+global.__values = function (o: any) {
```

Comment:
We need to override the typescript `__values` function because it doesn't handle multibyte strings properly. This will let us use `for...of` with multibyte strings.